runtime-image/bin/gen_keys.sh

```diff
@@ -30,10 +31,10 @@ RubenBridgewater=A48C2BEE680E841632CD4E44F07496B3EB3C1762
 RuyAdorno=108F52B48DB57BB0CC439B2997B01419BD92F80A
 ShelleyVohr=B9E2F5981AA6E0CD28160D9FF13993A75599653C
 
-keylist="${EvanLucas} ${GibsonFahnestock} ${JeremiahSenkpiel} ${ChrisDickinson} ${IsaacZSchlueter} ${ItaloACasas} ${JulienGilli} ${TimothyJFontaine} ${BethGriggs} ${ColinIhrig} ${DanielleAdams} ${JamesMSnell} ${MichaelZasso} ${MylesBorins} ${RichardLau} ${RodVagg} ${RubenBridgewater} ${RuyAdorno} ${ShelleyVohr}"
+keylist="${EvanLucas} ${GibsonFahnestock} ${JeremiahSenkpiel} ${ChrisDickinson} ${IsaacZSchlueter} ${ItaloACasas} ${JulienGilli} ${TimothyJFontaine} ${DanielleAdamsOld} ${BethGriggs} ${ColinIhrig} ${DanielleAdams} ${JamesMSnell} ${MichaelZasso} ${MylesBorins} ${RichardLau} ${RodVagg} ${RubenBridgewater} ${RuyAdorno} ${ShelleyVohr}"
 
 for key in ${keylist} ; do
-  gpg2 --keyserver pool.sks-keyservers.net --recv-keys ${key}
+  gpg2 --keyserver hkp://pgp.mit.edu:11371 --recv-keys ${key}
```

Comment:
Could you help me understand this change? 

Seems like pool.sks-keyservers.net is deprecated. How reliable and stable is hkp://pgp.mit.edu:11371?

Reply:
About pgp.mit.edu: https://pgp.mit.edu/about.html
About hkp: https://datatracker.ietf.org/doc/html/draft-shaw-openpgp-hkp-00
11371 is HKP default port

The future of nodejs release key next year is likely to use: https://github.com/nodejs/release-keys

For the upcoming 1-2 year, I think the mit host should be robust enough - it's unlikely they would need to shutdown their keyserver due to DMCA (like sks) given their nature as an education institution.